services/notify/src/handler.js

```diff
@@ -94,6 +94,9 @@ class Handler {
     // Load task definition
     let taskId = status.taskId;
     let task = await this.queue.task(taskId);
+    let artifact = await this.queue.latestArtifact(taskId, 'public/logs/live.log');
```

Comment:
In some installs, access to this log may be restricted, and it may contain sensitive information. So a few concerns here;
 - Check that `this.queue` has the correct credentials to represent a world-readable (e.g., email) view of the logs
 - If not, fail gracefully
 - Perhaps allow this to be disabled for a deployment, even if the log is "public" in terms of TC scopes (e.g., if the deployment is behind a firewall that would otherwise prevent the email recipients from reading the log)